src/style/_variables.scss

```diff
@@ -0,0 +1,9 @@
+$primary-background: black;
+$secondary-background: #E3D1BE;
+$heading-text-color: white;
+$secondary-text-color: #A86889;
+$first-accent-color: #F29024;
+$second-accent-color: #EB6825;
+$third-accent-color: #D56261;
+$fourth-accent-color: #057196;
+$fifth-accent-color: #3C9F88;
```

Comment:
This file looks great! Thanks for adding in our color theme and making them easier for us to use.